sdk/ai/azure-ai-resources/azure/ai/resources/operations/_azure_open_ai_deployment_operations.py

```diff
@@ -0,0 +1,80 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import uuid
+
+from typing import Iterable
+from azure.core.polling import LROPoller
+from azure.ai.ml import MLClient
+from azure.mgmt.resource import ResourceManagementClient
+from azure.mgmt.resource.resources.models import DeploymentMode
+
+from .._restclient._azure_open_ai._client import MachineLearningServicesClient
+from .._utils._deployment_utils import get_empty_azure_open_ai_arm_template
+from ..entities import AzureOpenAIDeployment
+
+class AzureOpenAIDeploymentOperations():
```

Comment:
Shall we just override the auto-generated operation class and overwrite the methods that need customization?

Reply:
as a future optimization, we can certainly try to figure it out. But the autogen client/methods need a way to access the rg/workspace scope since the autogen client is scoped to a specific subscription.